lib/OpenLayers/Tween.js

```diff
@@ -139,7 +158,10 @@ OpenLayers.Tween = OpenLayers.Class({
         this.time++;
         
         if (this.callbacks && this.callbacks.eachStep) {
-            this.callbacks.eachStep.call(this, value);
+            // skip frames if frame rate drops below threshold
+            if ((new Date().getTime() - this.startTime) / this.time <= 1000 / this.minFrameRate) {
```

Comment:
I think in this merge, a new issue shows up.
When switched to another window(focus is removed), the value of ```this.time``` will not increase,but the value of  ```new Date().getTime() - this.startTime``` will increase,so the caculate about frame cost is not  accurate.